docs/en/engines/table-engines/integrations/s3queue.md

```diff
@@ -118,7 +127,18 @@ Default value: `0`.
 
 Number of threads to perform processing. Applies only for `Unordered` mode.
 
-Default value: `1`.
+Default value: Number of CPUs or 16.
+
+### s3queue_parallel_inserts {#parallel_inserts}
+
+By default `processing_threads_num` will produce one `INSERT`, so it will only download files and parse in multiple threads).
+But this limits the parallelism, so for better throughput use `parallel_inserts=true`, this will allow to insert data in parallel.
```

Comment:
```suggestion
But this limits the parallelism, so for better throughput use `parallel_inserts=true`, this will allow to insert data in parallel, but keep in mind that it will result in higher number of generated data parts.
```

Reply:
Slightly adjusted add a few words about MergeTree and moved it into brackets, since it is more of a note.